common/authen.js

```diff
@@ -146,6 +149,7 @@
                         }
 
                         var onModalClose = function() {
+                            // TODO: Attach .catch w/ catchAll
```

Comment:
Same comment as above:
This is unnecessary because the outer `.catch()` that should be added to [line 181](https://github.com/informatics-isi-edu/chaise/blob/rec-errors/common/authen.js#L182) will catch an error from here.